content/en/docs/concepts/architecture/cri.md

```diff
@@ -29,10 +29,10 @@ runtime, which can be configured separately within the kubelet by using the
 `--image-service-endpoint` and `--container-runtime-endpoint` [command line
 flags](/docs/reference/command-line-tools-reference/kubelet)
 
-For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
+For Kubernetes v{{< skew currentPatchVersion >}}, the kubelet prefers to use CRI `v1`.
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.